pkg/cmd/codespace/common.go

```diff
@@ -31,23 +31,35 @@ type executable interface {
 	Executable() string
 }
 
+type progressIndicator interface {
+	StartProgressIndicatorWithLabel(s string)
+	StopProgressIndicator()
+}
+
+type statusChecker func(progressIndicator, codespaces.LiveshareApiClient, context.Context, *api.Codespace) error
```

Comment:
A Go convention is that if the function takes a context, it should be the first parameter. Can we move `context.Context` to the be first?

Reply:
agreed. missed this, sorry

Reply:
👍🏻  TIL thanks!